CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # AmpliPi Software Releases
 
+## Upcoming release
+* System
+  * Make update process properly report errors
+
+## 0.4.4
+* Web App
+  * Ensure that media device audio streams stop outputting audio when disconnected
+  * Limit max height of group creation dropdown
+  * Improve how marquee components look when not scrolling
+
```

Comment:
It appears we never updated the changelog for the released material in 0.4.4

Reply:
Yup. We did a couple of things our of order in the 0.4.4 release.